.github/workflows/binder.yml

```diff
@@ -0,0 +1,17 @@
+name: binder
+
+on: [push]
+
+# This job builds and caches the Docker image for binder on their infrastructure
```

Comment:
How long does binder store a cached docker image? This repo is almost never updated and I'm not sure if the short course is taught often enough to justify this...

Reply:
I don't know. This is very much an online learning experience :-) My limited personal experience is that the image stays cached at least for a few days. So I would fire off the server before I face the students....